controller/backup_target_controller.go

```diff
@@ -433,6 +433,13 @@ func (btc *BackupTargetController) reconcile(name string) (err error) {
 		}
 	}()
 
+	// clean up invalid backup volumes that are created during split-brain
+	// https://github.com/longhorn/longhorn/issues/11154
+	volumeBackupVolumeMap, err := btc.cleanupDuplicateBackupVolumes(backupTarget)
```

Comment:
Separate the `cleanupDuplicateBackupVolumes` into two functions, one is collecting `volumeBackupVolumeMap` and the other one is cleaning up the duplicates.

Reply:
Rename `volumeBackupVolumeMap` to `volumeBVMap`